Common/cpp/SharedItems/MutableValue.cpp

```diff
@@ -16,7 +16,7 @@ void MutableValue::setValue(jsi::Runtime &rt, const jsi::Value &newValue) {
     }
   };
   
-  if (module->isUIRuntime(rt)) {
+  if (RuntimeDecorator::isWorkletRuntime(rt)) {
```

Comment:
I don't think that RuntimeDecorator should have such methods. Maybe it will fit better with RuntimeManager?

Reply:
RuntimeDecorator is the one responsible for marking a runtime a "worklet runtime", why shouldn't it also be exposing a method that can determine this?

Reply:
You have convinced me.